File: templates/TwentyTen/UltraWebApplicationWithAssets/UltraWebApplicationWithAssets/UltraWebService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using ScriptCoreLib.Ultra.Library.Delegates;
using System.Diagnostics;

namespace UltraWebApplicationWithAssets
{
	public sealed class UltraWebService
	{
		public void GetTime(string x, StringAction result)
		{
			Debugger.Break();

			result(x + DateTime.Now);
		}
	}
}

File: templates/TwentyTen/VisualBasic/UltraWebApplicationWithAssets/UltraWebApplicationWithAssets/UltraWebService.vb
Imports ScriptCoreLib.Ultra.Library.Delegates

Public NotInheritable Class UltraWebService
    ' Methods
    Public Sub GetTime(ByVal x As String, ByVal result As StringAction)
        Debugger.Break()
        result.Invoke((x & DateTime.Now))
    End Sub

End Class


